include/irq.h

```diff
@@ -50,6 +50,19 @@
 /*************************** Types Declarations *******************************/
 /******************************************************************************/
 
+/**
+ * @enum irq_uart_event_e
+ * @brief Possible events for uart interrupt
+ */
+enum irq_uart_event_e {
```

Comment:
If we didn't need dependency on irq from uart.h, should this be here? 
(To put the events here was decided in a previous meeting, but in the current one, it was specified that we shouldn't have irq in uart)